foreign_cc/private/runnable_binary_wrapper.sh

```diff
@@ -19,6 +11,8 @@ source "$(grep -sm1 "^$f " "$0.exe.runfiles_manifest" | cut -f2- -d' ')" 2>/dev/
 { echo>&2 "ERROR: cannot find $f"; exit 1; }; f=; set -e
 # --- end runfiles.bash initialization v2 ---
 
+cd "${RUNFILES_DIR}"
+
 EXE=EXECUTABLE
 EXE_PATH=$(rlocation "${EXE#external/}")
```

Comment:
If `RUNFILES_DIR` isn't set to an absolute path, then the `EXE_PATH` may be incorrect after the `cd`. This can't easily be caught in tests since those `cd` to the `RUNFILES_DIR` anyway.

Reply:
Thanks @fmeum, I added the directory existence check, and made the `RUNFILES_DIR` absolute